internal/action/command.go

```diff
@@ -544,6 +545,15 @@ func setGlobalOptionNative(option string, nativeValue interface{}, write bool) e
 		}
 	}
 
+	if reflect.DeepEqual(config.GlobalSettings[option], nativeValue) {
```

Comment:
This is after we are done with locals, so it should be after the below comment `// ...if it's not local continue with the globals`?